sensors/magnetometersensor/magnetometerscalefilter.cpp

```diff
@@ -30,7 +30,7 @@
 MagnetometerScaleFilter::MagnetometerScaleFilter() :
         Filter<CalibratedMagneticFieldData, MagnetometerScaleFilter, CalibratedMagneticFieldData>(this, &MagnetometerScaleFilter::filter)
 {
-    factor = Config::configuration()->value("magnetometer/scale_coefficient", QVariant(300)).toInt();;
+    factor = Config::configuration()->value("magnetometer/scale_coefficient", QVariant(1)).toInt();
```

Comment:
hybris mag sensor values are proper values that incorporate scaling factor by the time it reaches here.